sources/packages/web/src/components/common/SearchStudents.vue

```diff
@@ -1,60 +1,66 @@
 <template>
   <v-form ref="searchStudentsForm">
     <content-group class="mb-8">
-      <v-row
-        ><v-col>
-          <v-text-field
-            label="Application number"
-            density="compact"
-            data-cy="appNumber"
-            variant="outlined"
-            v-model="appNumber"
-            @keyup.enter="searchStudents"
-            hide-details
-          />
+      <v-row>
+        <v-col cols="11" lg="11" md="10" sm="12">
```

Comment:
IMO I was expecting no need to use the cols at this level to achieve the responsiveness here. Now looking at how the code was I was able to understand what was forcing the text inputs to not break in multiple lines.
As a suggestion, I believe that a similar result can be achieved as with the below template.
```vue
<v-row>
        <v-col>
          <v-row class="search">
            <v-text-field
              label="Application Number"
              density="compact"
              data-cy="appNumber"
              variant="outlined"
              v-model="appNumber"
              @keyup.enter="searchStudents"
              hide-details
            />
            <v-text-field
              label="SIN"
              density="compact"
              data-cy="sin"
              variant="outlined"
              v-model="sin"
              @keyup.enter="searchStudents"
              hide-details
            />
            <v-text-field
              label="Given names"
              density="compact"
              data-cy="firstName"
              variant="outlined"
              v-model="firstName"
              @keyup.enter="searchStudents"
              hide-details
            />
            <v-text-field
              label="Last name"
              density="compact"
              data-cy="lastName"
              variant="outlined"
              v-model="lastName"
              @keyup.enter="searchStudents"
              hide-details
            />
          </v-row>
        </v-col>
        <v-col cols="auto">
          <v-btn
            color="primary"
            class="p-button-raised"
            data-cy="searchStudents"
            @click="searchStudents()"
          >
            Search
          </v-btn>
        </v-col>
      </v-row>
```

The class used is the below one:
```scss
<style lang="scss">
.search {
  padding: 5px;
  .v-input {
    min-width: 120px;
    margin: 5px;
  }
}
</style>
```

Reply:
Thanks for the suggestion, but with the code above depending on the width the layout can be weird like this:
![image](https://user-images.githubusercontent.com/78114138/232167320-5a988372-d98d-453a-b269-db97ceb53ce3.png)

Reply:
If the "weird" part is related to the application filed, it can be adjusted with the width, as shown below. It the "weird" is because the last name takes up all the space then it will need more work.
Either way, the idea was to suggest a different approach where there was no need to have every single input wrapped with a `v-col`. If the current solution is working no need to spend more time on this.

![image](https://user-images.githubusercontent.com/61259237/232169023-96ad2983-8f46-4010-be8a-93a643a97e44.png)